views/pages/StudiesPage/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+
+import { Box } from '@mui/material'
+
+import { SORT_DIRECTION } from '../../../constants'
+import api from '../../api'
+import PageHeader from '../../components/PageHeader'
+import useTableSort from '../../hooks/useTableSort'
+import StudiesTable from '../../tables/StudiesTable'
+
+const site = 'site'
+const study = 'study'
+
+const StudiesPage = () => {
+  const [studies, setStudies] = useState([])
+  const { onSort, sortDirection, sortBy } = useTableSort('study')
+  const loadStudies = async () => {
+    const sortParams = {
+      ...(sortBy ? { sortBy: sortBy === site ? study : sortBy } : {}),
+      ...(sortDirection ? { sortDirection } : {}),
+    }
+    const data = await api.studies.loadAll({ sort: sortParams })
+
+    setStudies(data)
+  }
+  const handleRequestSort = (_event, property) => {
+    const isAsc = sortBy === property && sortDirection === SORT_DIRECTION.ASC
```

Comment:
do we need to check the property?

Reply:
good catch 
[6389301](https://github.com/AMP-SCZ/dpdash/pull/723/commits/638930192c05d8f442f77f5a8641cc1a1ca13ab7)